### ISSUE TITLE
"Click here to reload" not clickable

### ISSUE THREAD

**Issue by Epse** [pierr3/TrackAudio#232]
After updating and changing some settings (PTT and mini mode transparency), I received a prompt in the bottom of the main window, "Click here to reload to apply changes".
The cursor does not change when hovering and clicking it appears to have no effect

Log file appears not to contain anything related to clicking that link, but here it is for completeness.
[trackaudio.log](https://github.com/user-attachments/files/17902917/trackaudio.log)

**Comment by GeorgeBarlow** [comment]
Yes this was an extreme oversight on my behalf, it has been already fixed. Pierre and I were discussing on adding a hotfix release to b4, should be out shortly.
